src/sds.c

```diff
@@ -1338,6 +1315,21 @@ sds sdstemplate(const char *template, sdstemplate_callback_t cb_func, void *cb_a
     return NULL;
 }
 
+/* Returns 1 if s contains the character in p, otherwise returns 0. */
+int sdscontains(const sds s, const char *p, size_t plen) {
```

Comment:
sorry for thinking of it just now, but i realize there's no reason to introduce a specialized function, and not to use `strpbrk` instead.